lib/js/src/session/_SdlSession.js

```diff
@@ -168,6 +170,25 @@ class _SdlSession {
         this._sdlSessionListener.onAuthTokenReceived(authToken, this._sessionId);
     }
 
+    /**
+     * A way to determine if VehicleType already received
+     * @returns {Boolean} Return true if received
+     */
+    didReceiveVehicleType() {
```

Comment:
There needs to be a space before the function parentheses on line 177. Use the command `npm run lint` to confirm.

Reply:
Fixed